### ISSUE TITLE
In certain situations, nvda does not announce the open menu when pressing applications in chrome

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#17155]
### Steps to reproduce:
open chrome
Now press start + d to go to the desktop, put the mouse focus on the desktop by pressing nvda + shift + m, press alt tab to the chrome window and press the applications key.
### Actual behavior:
When you move with the arrows, nvda continues to navigate through the document in navigation mode.
### Expected behavior:
nvda should announce the opening of the menu and navigate through the items.
A sighted person has informed me that the menu is open.
This appears to be intermittent, the steps like this seem to force the problem to be reproduced reliably, but it occurs on other occasions where I have not established what happens to start.
Ultimately it is an intermittent problem.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-33940,20bdb39e

#### Windows version:
10.0.19045.4842
#### Name and version of other software in use when reproducing the issue:
chrome.exe, Google Chrome 128.0.6613.120

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[mouse.txt](https://github.com/user-attachments/files/16949960/mouse.txt)

**Comment by CyrilleB79** [comment]
I have just noticed this issue this morning. I cannot tell if it's due to a Chrome update or to NVDA.
I am following NVDA's beta channel, currently running 2024.4beta3.
Chrome version is 128.0.6613.120

And the issue is also described on [NVDA mailing list](https://nvda.groups.io/g/nvda/topic/issue_with_nvda_and_google/108388336).

**Comment by XLTechie** [comment]
Could this be similar to #15300 or #15309? @LeonarddeR worked on those Edge problems I think. I can't tell from a quick read, whether the same things are possibly going on here. I don't know what's normal for Chrome.

**Comment by Keithcat1** [comment]
I have noticed this too, only works if aI press application + right arrow.
Aparrently Chrome broke some stuff yesterday by enabling some kind of UIA feature by default even though it had bugs. A solution appears to be to invole chrome.exe with the following command line parameters:
```--disable-features=UiaProvider
```

**Comment by Adriani90** [comment]
cc: @aleventhal, @dsexton I don't know for what exactly I should create a bug on Chromium in this regard, could you please have a look? It seems it's a broader issue accoring to https://github.com/nvaccess/nvda/issues/17155#issuecomment-2344077050

Or did anyone already created a bug on Chromium bug tracker?

**Comment by aleventhal** [comment]
Thanks for notifying us. We know what it is. It's an experiment with UIA.
If you run chrome.exe --disable-features=UiaProvider it will fix the issue.
However, we're working on disabling the experiment via a remote kill switch
right now, so the problem should go away shortly.

**Comment by Adriani90** [comment]
Is this solved now in Chrome 129.0.6668.70/.71 with NVDA 2024.4 Beta? Can we close this issue?

**Comment by CyrilleB79** [comment]
It did not occur anymore on my side for some days. OK for me to close this issue.

**Comment by Adriani90** [comment]
Thanks for the quick reply, Let's close this for now. if anyone experiences this again, we can reopen and restart the loop again.
